### ISSUE TITLE
Problem with the Sm 1.9 

### ISSUE THREAD

**Issue by ZeitZumKiffen** [Kxnrl/Mapchooser-Redux#10]
It works but the system is little bit weird :/ sm version is 1.9

mapchooser_redux.cfg
https://pastebin.com/v6L9dVp0

maplister_redux.cfg
https://pastebin.com/MxJRBsVA

the votesystem (how i can delete the white text) and the revote is doesn't work
https://i.gyazo.com/4286821ba6d0d3615bd560349b9b48bc.jpg

same with timelimit,nominate and votemap, the menu is not coming
https://i.gyazo.com/c53abe18f5fe5bca835649840095bf1a.png

how i can fix that? :/ but otherwise everything looks good, thanks

**Comment by ThanhByKyTa** [comment]
...cfg/sourcemod/mapchooser/mapchooser_redux.cfg

mcr_maps_history_count "xx"

**Comment by ZeitZumKiffen** [comment]
> mcr_maps_history_count "xx"

Hey, Thank you for your reply but thats the cooldown for the maps

> // How many maps cooldown
// -
// Default: "15"
// Minimum: "1.000000"
// Maximum: "300.000000"
mcr_maps_history_count "3"

I wanted to have it so that after 60 seconds rtv goes, and that when rtv (voteend) comes an almost change the map.

**Comment by ThanhByKyTa** [comment]
@ZeitZumKiffen 
Sorry i can't help you but @Kxnrl help you soon :)

**Comment by ZeitZumKiffen** [comment]
i hope, but thanks :D

**Comment by Kxnrl** [comment]
> I wanted to have it so that after 60 seconds rtv goes, and that when rtv (voteend) comes an almost change the map. 

I don't know what do you mean.??

**Comment by ZeitZumKiffen** [comment]
> > I wanted to have it so that after 60 seconds rtv goes, and that when rtv (voteend) comes an almost change the map.
> 
> I don't know what do you mean.??

Hey @Kxnrl, after a map change you should be able to enter rtv only from 1 min + nominate and if the map was used with rtv, it should be changed immediately....maybe is it possible to add a cvar? or a cfg in cfg/sourcemod/mapchooser

**Comment by Kxnrl** [comment]
!map use with !rtv?

**Comment by ZeitZumKiffen** [comment]
No, It is only after a while allowed to enter rtv on your script (i think is 300 Seconds) and if the vote was used with rtv, it should change immediately after the voteend :D

**Comment by ZeitZumKiffen** [comment]
you are still alive? or do you still not understand the question?

**Comment by Kxnrl** [comment]
i don't understand.

**Comment by ZeitZumKiffen** [comment]
1.
Do you know the rockthevote from sourcemod (original) ? When you type rtv, the menu is showing up, so you vote a MAP after the all have voted, the Map change comes immediately (ONLY BY RTV) thats not on your script included, maybe you can add a cvar.

2.
You can only use rtv after a certain time, how do I change it to 1 minute? i think in your script is it 300 seconds (5 min)

so i hope you understand this :D

**Comment by Wh1t3Rose** [comment]
I am going to clarify this since I am from the United States I and I can speak proper English.

Right now the way your plugins works is that when your rtv in chat, the vote menu appears, Then after everyone votes for a map it doesn't actually change to that map.

What is supposed to happen is that when you rtv and the vote ends, the map should IMMEDIATELY change to the voted map and NOT when the round timer ends,

**Comment by Gelmo** [comment]
Have you had an opportunity to look into this?

**Comment by Kxnrl** [comment]
ofc, but almost no time, i live in china and the COVID-19....

**Comment by Dism0raL** [comment]
Need help
In now version "mcr_maps_history_count" doesn't exist in cfg.

I have a problem with maps cooldown. All my maps in cooldown.
This can be seen in the nomination menu.
![2](https://user-images.githubusercontent.com/41348155/77237379-c0349180-6bd8-11ea-9ec4-6f8394304333.jpg)

And when the map ends, it gives you 0 - 1 maps to choose from. 
I need a choice of all the cards. Without cooldown.

I use cfg (If I'm suddenly wrong in it - correct):
mcf_shuffle_require_maps "5"
mcr_include_desctag "0"
mcr_include_nametag "0"
mcr_map_extend_times "1"
mcr_partyblock_enabled "0"
mcr_rectplayed_interval "1" - I tried "-1", and "0"
mcr_rectplayed_ltp_mtpl "0.5"
mcr_refund_credits_ratio "0.6"
mcr_timer_hud_location "2"
mcr_votemenu_shuffle "0"

SourceMod (1.10.0.6474)
Metamod:Source 1.10.7-dev
